ChromecastSampleApp/ChromecastSampleApp/Players/PlayerViewController.m

```diff
@@ -158,6 +158,7 @@ - (void)configureCastPlaybackViewBasedOnItem:(OOVideo *)item {
     NSString *videoDescription = item.itemDescription;
 
     dispatch_sync(dispatch_get_main_queue(), ^{
+      [[self.castPlaybackView subviews] makeObjectsPerformSelector:@selector(removeFromSuperview)];
```

Comment:
Sure, I assume we could more manually/explicitly tell views to go away individually? I guess I don't grok what might be wrong with this approach here, tho. :-) The goal is always to clean out old ones, and this does that. And it is not hardcoded against any list of things. So it cannot go wrong in the sense of forgetting to duplicate add/remove code for some new/old view.